charts/kong-plugins/values.yaml

```diff
@@ -0,0 +1,7 @@
+# Default values for kong-plugins.
+
+namespace: kong
```

Comment:
I am not sure this is needed tbh because helm charts are normally installed with tools that impose a namespace.